Trimble.Modus.Components/Controls/TMSwitch/TMSwitch.xaml.cs

```diff
@@ -0,0 +1,210 @@
+using System.Windows.Input;
+using Trimble.Modus.Components.Constant;
+using Trimble.Modus.Components.Enums;
+using Trimble.Modus.Components.Helpers;
+
+namespace Trimble.Modus.Components;
+
+public partial class TMSwitch : ContentView
+{
+    protected EventHandler<TMSwitchEventArgs> _clicked;
+    private bool _switchSelected;
```

Comment:
Do we need this property at all? Can't we use the `IsToggled` itself?

Reply:
Yes ,we can and its changed